Python/path.txt

```diff
@@ -0,0 +1,5 @@
+Data/messages/Falcon/
+ASIST_data_study_id_000001_condition_id_000005_trial_id_000009_messages.json
```

Comment:
Changing the name of the json file will be time consuming when we have to process multiple files. It would be preferable to use naming conventions to identify all files in the folder that follow a similar pattern, and process them automatically. For example, all files ending in "messages.json" should be processed.

Reply:
The survey file is the xlsx one from the data directory.

Reply:
> Changing the name of the json file will be time consuming when we have to process multiple files. It would be preferable to use naming conventions to identify all files in the folder that follow a similar pattern, and process them automatically. For example, all files ending in "messages.json" should be processed.

Thanks, it's in process.